practice-one/src/index.html

```diff
@@ -42,5 +44,72 @@
         </nav>
     </header>
     <!-- End Header -->
+    <!-- Start footer -->
+    <footer class="footer-section">
+        <div class="footer-container">
+            <div class="footer-item-top">
+                <div class="item-top">
+                    <h4 class="text-primary text-white text-title-footer">contact us</h4>
+                    <address class="contact-us">
+                        <p class="text-primary description text-white opacity break">2300 W Arbutus Dr Citrus Springs, FL 34434, USA</p>
+                        <div>  
+                            <div class="text-primary description text-white opacity">+235 793-891-7938</div>
```

Comment:
This is phone number, right?

Reply:
Yes, this is phone number. Should I create animations  such as underline and clickable when we hover it?

Reply:
Please help to check attr href of a tag

Reply:
I fixed it
![image](https://github.com/HoangLiarPhuongLe/html_css_training/assets/110210274/6b14993b-3236-4076-8757-23549ca1de26)